test/serializer/abstract/ArrayInitializer.js

```diff
@@ -0,0 +1,11 @@
+(function() {
```

Comment:
Prepend test case with a comment like this (enforced by the test runner):

// does contain:[11, 22